components/steplist/index.css

```
@@ -322,4 +322,12 @@ governing permissions and limitations under the License.
     margin-inline-end: 0;
     margin-inline-start: calc(-1 * calc(var(--mod-steplist-marker-hitArea, var(--spectrum-steplist-marker-hitArea)) / 2));
   }
-}
\ No newline at end of file
+}
+
+@media (forced-colors: active) {
+	.spectrum-Steplist {
+    --highcontrast-steplist-complete-marker-background-color: ButtonText;
+    --highcontrast-steplist-current-marker-color: SelectedItem;
+    --highcontrast-steplist-current-marker-color-key-focus: Highlight;
+  }
+}
```

```suggestion
@media (forced-colors: active) {
  .spectrum-Steplist {
    --highcontrast-steplist-complete-marker-background-color: CanvasText;
    --highcontrast-steplist-current-marker-color: Highlight;
    --highcontrast-steplist-current-marker-color-key-focus: Highlight;
  }

  @supports (color: SelectedItem){
    .spectrum-Steplist {
      --highcontrast-steplist-current-marker-color: SelectedItem;
    }
  }
}
```
Two suggestions here:
- I don't think `ButtonText` is the best system color for the default marker unless it's paired with background using `ButtonFace`; I think `CanvasText` would be more appropriate?
- `SelectedItem` is still pretty new to the spec and browser support, so I'd recommend an `@supports` here, with a fallback to `Highlight`.